tests/testthat/test-tar-terra-sprc.R

```diff
@@ -16,6 +16,16 @@ targets::tar_test("tar_terra_sprc() works", {
         ) * z,
         projection
       )
+      # test to ensure metadata comes along.
+      # e.g., layer names, units, time, variables
+      # The snapshots for these tests do not include those metadata
+      # so this may be as easy as assigning some of these attributes in the
+      # test SPRC/SDS and adding those fields in the snapshot.
+      terra::units(rast_elev_scale) <- "m"
+      terra::varnames(rast_elev_scale) <- "elev"
+      terra::longnames(rast_elev_scale) <- "really-long-name"
```

Comment:
these won't be saved due to limitations in `terra`

Reply:
Got it, thanks!